src/beautifiers/js-beautify.coffee

```diff
@@ -31,7 +31,7 @@ module.exports = class JSBeautify extends Beautifier
     return new @Promise((resolve, reject) =>
       try
         switch language
-          when "JSON", "JavaScript", "JSX"
+          when "JSON", "JavaScript", "JSX", "TSX"
```

Comment:
`TSX` is incorrect, since the `language` in this case would actually be `JSX` from `languages/jsx.coffee`.